docs/editor_manual/administrator_tasks/managing_workflows.rst

```diff
@@ -0,0 +1,43 @@
+Managing Workflows
+==================
+
+Workflows allow you to configure how moderation works on your site. Workflows are sequences of tasks, all of which must be approved
+before the workflow completes (by default, this results in the publication of the page, but depends on your site settings TODO: add settings link).
```

Comment:
Let's make sure these TODOs don't get forgotten :-)

Reply:
Weirdly, I was sure I'd fixed this one - thanks for catching, I'll add back in.

Reply:
Wait - these actually were done. Somehow the commit for these was lost, I'll try and find it again!

Reply:
Cherry-picked over now :)

Reply:
Ah, sorry - they're on your original PR, at https://github.com/wagtail/wagtail/pull/5856/commits/35811da42666396795a1cccc81f25189964c1c66 and https://github.com/wagtail/wagtail/pull/5856/commits/bea480aac759f64c9a74bb3e5cf503c215d26f77 . Must have missed those somehow while rebasing!